src/main/java/com/synopsys/integration/detect/configuration/DetectProperties.java

```diff
@@ -312,6 +312,14 @@ private DetectProperties() {
             .setGroups(DetectGroup.BINARY_SCANNER, DetectGroup.SOURCE_SCAN)
             .build();
 
+    public static final NullablePathProperty DETECT_CONTAINER_SCAN_FILE =
+        NullablePathProperty.newBuilder("detect.container.scan.file.path")
+            .setInfo("Container Scan Target", DetectPropertyFromVersion.VERSION_4_2_0)
```

Comment:
Thanks for catching that. Since this is planned for Detect v8.11.0, I'll add `VERSION_8_11_0("8.11.0");` to the `DetectPropertyFromVersion` enum and update this line.